Gifski/SavePanelAccessoryViewController.swift

```diff
@@ -83,4 +186,61 @@ final class SavePanelAccessoryViewController: NSViewController {
 		qualitySlider.doubleValue = defaults[.outputQuality]
 		qualitySlider.triggerAction()
 	}
+
+	private func scalingTextFieldTextDidChange(_ textField: NSTextField) {
+		let valid: Bool
+		if textField == widthTextField {
+			let width = CGFloat(Double(self.widthTextField.stringValue) ?? 0.0)
+			valid = resizableDimensions.validate(newWidth: width)
+			resizableDimensions.resize(usingWidth: width)
+		} else if textField == heightTextField {
+			let height = CGFloat(Double(self.heightTextField.stringValue) ?? 0.0)
+			valid = resizableDimensions.validate(newHeight: height)
+			resizableDimensions.resize(usingHeight: height)
+		} else {
+			return
+		}
+
+		if !valid {
+			textField.shake()
+		}
+		dimensionsUpdated()
+		predefinedSizesDropdown.selectItem(at: 0)
+	}
+}
+
+extension SavePanelAccessoryViewController: NSTextFieldDelegate {
+	func controlTextDidChange(_ obj: Notification) {
+		guard let textField = obj.object as? NSTextField else {
+			return
+		}
+		scalingTextFieldTextDidChange(textField)
+	}
+
+	func control(_ control: NSControl, textView: NSTextView, doCommandBy commandSelector: Selector) -> Bool {
```

Comment:
See https://github.com/sindresorhus/gifski-app/pull/49#discussion_r248931694

Reply:
Oh, missed it. Will do for sure.